# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/webauthn/tests/mochitest.toml
# Commit: e1dc56ae7277
# Full Hash: e1dc56ae7277b29a3ecbd5ec3cd8612a3697b57a
# Author: John M. Schanck <jschanck@mozilla.com>
# Date: 2025-06-06 08:52:03
# Description:
#   Bug 1969341 - clean up WebAuthn transaction state before resolving promises. r=djackson
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251966
# ==============================================================================

diff -r 81f3e9043996 -r e1dc56ae7277 dom/webauthn/tests/mochitest.toml
--- a/dom/webauthn/tests/mochitest.toml	Fri Jun 06 02:05:23 2025 +0000
+++ b/dom/webauthn/tests/mochitest.toml	Fri Jun 06 04:20:30 2025 +0000
@@ -17,6 +17,10 @@
   "security.webauthn.enable_macos_passkeys=false",
 ]
 
+["test_bug1969341.html"]
+fail-if = ["xorigin"] # NotAllowedError
+run-if = ["os != 'win'"] # Bug 1718296 (Windows 10 1903+ has its own window and U2F that we cannot control with tests.)
+
 ["test_webauthn_abort_signal.html"]
 fail-if = ["xorigin"]
 run-if = ["os != 'win'"] # Bug 1718296 (Windows 10 1903+ has its own window and U2F that we cannot control with tests.)